Contribute/includes/note-docsitelocfeedback.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> The email template available at [aka.ms/DocSiteLocFeedback](https://aka.ms/DocSiteLocFeedback) handles localization-related feedback. We do not offer public Github repositories for localized content feedback with the exception of azure-docs, which is open for community contribution for 6 additional languages (Simplified Chinese, French, German, Japanese, Korean, Spanish). For all other repositories and localizations, use the [email template](https://aka.ms/DocSiteLocFeedback) — which receives your input in any language — to report translation issues or to provide technical feedback. 
```

Comment:
@WilliamDAssafMSFT William, I would suggest starting this include not out more directly with the problem and then the solution of the template. Something like this:

```suggestion
> Most localized documentation does not offer the ability to edit or provide feedback through GitHub. To provide feedback on localized content, use the email template available at [aka.ms/DocSiteLocFeedback](https://aka.ms/DocSiteLocFeedback). There is one exception for Azure documentation. There are public localized GitHub repositories for azure-docs that are open for community contribution for 6 languages: Simplified Chinese, French, German, Japanese, Korean, and Spanish. For all other languages and content sets, use the [email template](https://aka.ms/DocSiteLocFeedback)—which receives your input in any language—to report translation issues or to provide technical feedback. 
```

Reply:
I like it